tectonic/dpx-fontmap.c

```diff
@@ -1281,7 +1281,7 @@ strip_options (const char *map_name, fontmap_opt *opt)
     return font_name;
 }
 
-#if  DPXTEST
+#ifdef  DPXTEST
```

Comment:
This whole chunk of code can just be deleted because we are not in fact ever going to `#define DPXTEST`.